### DISCUSSION TITLE
Latest version of scap-security-guide in RHEL repos

### DISCUSSION THREAD

**Discussion by bmphbWV0** [ComplianceAsCode/content#11352] [General]
Hello,
Have you planned to push the latest rpm package (0.1.70) of scap-security-guide in RHEL (8-9) repositories ?

**Comment by dahaic** [comment]
Hello @njt-cla. For RHEL-specific questions like what will be released when, please either open new Jira ticket at https://issues.redhat.com/projects/RHEL/ or ask via customer case https://access.redhat.com/support/cases/#/case/new/get-support?caseCreate=true You should get faster response that way :)

And to answer your question - very likely candidate for RHEL release is version 0.1.72.
